Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -282,10 +282,16 @@ final class FileManagerTests : XCTestCase {
             try $0.createDirectory(atPath: "create_dir_test2/nested2", withIntermediateDirectories: true)
             XCTAssertEqual(try $0.contentsOfDirectory(atPath: "create_dir_test2").sorted(), ["nested", "nested2"])
             XCTAssertNoThrow(try $0.createDirectory(atPath: "create_dir_test2/nested2", withIntermediateDirectories: true))
+            
+            #if os(Windows)
+            try $0.createDirectory(atPath: "create_dir_test3\\nested", withIntermediateDirectories: true)
```

Comment:
This is an example of a previous API that used to fail without this change. Previously, this API called `deletingLastPathComponent` on the provided path, which would return an empty string and therefore fail to create the parent `create_dir_test3` directory. Now it correctly creates `create_dir_test3` followed by `create_dir_test3/nested`